winpr/include/winpr/sspi.h

```diff
@@ -679,6 +689,42 @@ typedef struct _SEC_WINNT_AUTH_IDENTITY SEC_WINNT_AUTH_IDENTITY;
 
 #endif /* _AUTH_IDENTITY_DEFINED */
 
+#ifndef SEC_WINNT_AUTH_IDENTITY_VERSION
+#define SEC_WINNT_AUTH_IDENTITY_VERSION 0x200
+
+typedef struct _SEC_WINNT_AUTH_IDENTITY_EXW
```

Comment:
Better not use the `_SEC` struct names, they spill lots of warnings with modern compilers (reserved identifier)

Reply:
Well I can rename the one I've added, but the other ones are windows things.

Reply:
@hardening yes, they use an old style type declaration but you can omit the name in a  `typedef struct name {} typename` type definition.